### PR TITLE
Reveal the invalid path

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: src/Test/Utilities/Portable/FX/ImmutableArrayTestExtensions.cs [hunk 1]
```diff
@@ -24,8 +22,15 @@ public static class ImmutableArrayTestExtensions
...
-            using var fileStream = new FileStream(path, FileMode.Create, FileAccess.Write, FileShare.Read, BufferSize);
-            WriteToStream(bytes, fileStream);
+            try
+            {
+                using var fileStream = new FileStream(path, FileMode.Create, FileAccess.Write, FileShare.Read, BufferSize);
+                WriteToStream(bytes, fileStream);
+            }
+            catch (ArgumentException e)
+            {
+                throw new ArgumentException($"Path 'path'", e);
```
Comment (jaredpar) [inline src/Test/Utilities/Portable/FX/ImmutableArrayTestExtensions.cs:32]:
```suggestion
                throw new ArgumentException($"Path {path}", e);
```

Comment (sharwell) [reply src/Test/Utilities/Portable/FX/ImmutableArrayTestExtensions.cs:32]: Sigh

**Comment by jaredpar** [comment]
Always fun when your PR to track down a flaky failure hits a different flaky failure: `   Microsoft.CodeAnalysis.CSharp.UnitTests.Emit.EndToEndTests.Constraints`.

If you want drop that down by 20 clicks in this PR and I can approve. If not I will try and get a PR out for that later today.

**Comment by jaredpar** [comment]
Merging as this is a test only change that is meant to help us track down other VS reliability issues.

**Pull request merged**